tests/test_models/test_engine/test_db_storage.py

```diff
@@ -86,3 +86,11 @@ def test_new(self):
     @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
     def test_save(self):
         """Test that save properly saves objects to file.json"""
+
+    @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
+    def test_get(self):
+        """Test that get returns the correct object"""
```

Comment:
user = User(name="User1")
        user.save()
        self.assertEqual(models.storage.get("User", user.id), user)

Reply:
Sure I had modified the file and made these changes